dashboard/modules/reporter/reporter_agent.py

```diff
@@ -22,6 +22,7 @@
 import psutil
 
 logger = logging.getLogger(__name__)
+routes = dashboard_utils.ClassMethodRouteTable
```

Comment:
`routes` is for RESTful APIs, but i can't find any routes usages in `reporter_agent`.

Reply:
Good catch, it shouldn't be there, thanks.

Reply:
ping

Reply:
So bizarre, I must have pushed these changes to a different branch or something, I remember addressing these this morning.

Reply:
Fixed